lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
+    CY_ID = 'wpcc'
```

Comment:
The English and Welsh IDs can be found in the description for this [story here](https://moneyadviceservice.tpondemand.com/entity/8191).

```
EN: https://www.moneyadviceservice.org.uk/en/tools/workplace-pension-contribution-calculator

CY: https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-cyfraniadau-pensiwn-gweithle
```